__tests__/switch-angular-builder.test.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  post10_1LibraryBuilder,
+  pre10_1LibraryBuilder,
+  replaceLibrariesBuildBuilder
+} from '../src/replace-libraries-build-builder';
+import {mixedAngularJson} from './angular-json/mixed-angular-json';
+import {post10_1LibraryBuilderWorkspace} from './angular-json/post10_1-library-builder-workspace';
+import {pre10_1LibraryBuilderWorkspace} from './angular-json/pre10_1-library-builder-workspace';
+
+describe(replaceLibrariesBuildBuilder.name, () => {
+  [
+    '6.0.x',
+    '6.1.x',
+    '7.0.x',
+    '7.1.x',
+    '7.2.x',
+    '8.0.x',
+    '8.1.x',
+    '8.2.x',
+    '9.0.x',
+    '9.1.x',
+    '10.0.x'
+  ].forEach(version => {
+    test(`when using the version ${version} it should replace all libraries build builder with ${pre10_1LibraryBuilder}`, () => {
+      const actual = replaceLibrariesBuildBuilder(version, mixedAngularJson);
+
+      expect(actual).toEqual(pre10_1LibraryBuilderWorkspace);
```

Comment:
> (Lars) suggestion: Make the assertion more narrow. We're only interested in one very specific change which isn't apparent from this assertion and requires reading hundreds of lines of configuration and spotting a few lines that differ.

⬆️

Reply:
I already remove everything from those files that weren't necessary. However I want to test that only the right builder is modified, that's why I left other projects without a build trigger.

Reply:
What I mean is let the test show that the builder name is what's changed. This is not currently clear in the test body, only in the test description. Assert on the build name of a library `build` architect target, not the entire workspace configuration.